examples/CMakeLists.txt

```diff
@@ -18,10 +18,13 @@ foreach(example_file ${examples})
     get_filename_component(example ${example_file} NAME_WE)
     add_executable(${example} ${example_file})
     target_link_libraries(${example} libodil)
+    install(TARGETS ${example} RUNTIME DESTINATION bin)
```

Comment:
I'd rather keep the examples out of the installed resources: for the command-line interaction, we now have an full [application](https://github.com/lamyj/odil/tree/master/applications/odil)